src/client/Components/Lists/JobsList.js

```diff
@@ -98,7 +98,7 @@ class JobsList extends React.Component {
 					<img className="status" src={icon} />
 				</a>
 				<div className="details">
-					<p><a href={`/jobs/${job.job_id}`}>{job.job_number}</a> | <a href={`/topics/${job.job_topic}`}><span className="topicgroup">{job.job_topic}</span></a>, <a href={`/groups/${job.group_name}`}><span className="topicgroup">{job.group_name}</span></a></p>
+					<p><a href={`/jobs/${job.job_id}`}>{/*job.job_number*/ job.job_topic}</a> | <a href={`/topics/${job.job_topic}`}><span className="topicgroup"> #{/*job.job_topic*/job.job_number}</span></a>, <a href={`/groups/${job.group_name}`}><span className="topicgroup">{job.group_name}</span></a></p>
```

Comment:
Watch out - Now if you click the topic it brings you to the job page and if you click the job number it brings you to the topic page. Swapping the `href` of the two <a> tags should fix this.
Also, maybe the group should come before the job number?